src/permission-ng/permission-ng.js

```diff
@@ -85,8 +85,8 @@
     });
   }
 
-  angular
+  module.exports = angular
     .module('permission.ng', ['permission', 'ngRoute'])
-    .run(run);
+    .run(run).name;
 
-}());
+}(module || {}));
```

Comment:
Your right about that one, however then `var module = module || {}` cannot be defined in the argument value but needs to be defined in the first line of the file.